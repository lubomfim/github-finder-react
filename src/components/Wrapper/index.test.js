import React from 'react';
import { render, screen } from '@testing-library/react';
import { Wrapper } from '.';

const ChildComponent = () => {
  return <div>Child Component</div>;
};

const WrapperComponent = () => {
  return (
    <Wrapper>
      <ChildComponent />
    </Wrapper>
  );
};

it('should render child component', () => {
  render(<WrapperComponent />);

  const child = screen.getByText('Child Component');
  expect(child).toBeInTheDocument();
});

it('should render correctly', () => {
  const { asFragment } = render(<WrapperComponent />);
  expect(asFragment(<WrapperComponent />)).toMatchSnapshot();
});
