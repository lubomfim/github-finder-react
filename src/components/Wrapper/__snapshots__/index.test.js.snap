// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<DocumentFragment>
  .c0 {
  background-color: #FFF;
  height: 100vh;
  max-width: 95%;
  margin: 0 auto;
}

<div
    class="c0"
  >
    <div>
      Child Component
    </div>
  </div>
</DocumentFragment>
`;
